### PR TITLE
PB-352 Add sdk config

### PR DESCRIPTION
**Pull request by Robert-Steiner**
### References

Follow up task to complete [PB-352](https://xainag.atlassian.net/browse/PB-352)

### Summary

* Added participant config

### Are there any open tasks/blockers for the ticket?

No

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [x] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "pytest==5.3.2",  # MIT license
     "pytest-cov==2.8.1",  # MIT
     "pytest-watch==4.2.0",  # MIT
-    "xain-sdk==0.4.0",  # Apache License 2.0
+    "xain-sdk @ git+https://github.com/xainag/xain-sdk.git@development#egg=xain_sdk-0.3.0",  # Apache License 2.0
```
Comment (little-dude) [inline setup.py]: it seems weird that xain-sdk 0.4 has been released but we create an egg with version 0.3 from the development branch. Does it matter at all?

Comment (janpetschexain) [inline setup.py]: shouldn't this point to `egg=xain_sdk-0.5.0`?

Comment (janpetschexain) [reply setup.py]: git "unfreshness" strikes again, sorry for duplicate.

Comment (Robert-Steiner) [reply setup.py]: We have tried to this once but it seems to be that pip can't figure out if something was updated on a git repository. That means, if we change the version number, pip won't clone the the repository again. Nevertheless, I agree it seems weird. I will change it to `0.5.0`.

**Pull request merged**